backend/.env.example

```diff
@@ -1,12 +0,0 @@
-PORT=8000
-
-MONGODB_URI=""
-
-JWT_EMAIL_VERIFICATION_SECRET=""
-
-SMTP_HOST=""
-SMTP_PORT=""
-SMTP_USER=""
-SMTP_PASSWORD=""
-
-CLIENT_URL=""
```

Comment:
Could you please return this file?
You can remove variables that are no longer used